lib/v2/lsnu/maintainer.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    '/lsnu/jiaowc': ['nyaShine'],
```

Comment:
```suggestion
    '/jiaowc/tzgg/:category?': ['nyaShine'],
```
https://github.com/DIYgod/RSSHub/blob/a8450f8561c694dc9de5dae9ca3b43051c7dceb7/lib/v2/lsnu/router.js#L2
They should be the same.